README.md

```diff
@@ -13,6 +13,8 @@
 
 # Configuring Your Local Environment
 
+You can deploy the stack from the following repository: [AWS VPC Creator](https://github.com/SofarDatas/aws-vpc-creator). After deployment, you can use the VPC_ID obtained from there as an input parameter.
```

Comment:
**Suggestion:** Enhance the README by providing more detailed instructions on how to obtain and use the VPC_ID from the AWS VPC Creator repository. This can include steps to navigate the repository, run the deployment, and locate the VPC_ID in the output.
> 🍄  **Medium Priority** 
 > [Category: **`documentation`**, Importance: **`7`**]
> **Reason:** This suggestion significantly improves the documentation, making it easier for users to correctly set up and use the VPC_ID, which is essential for the application's deployment.

```suggestion
To obtain the VPC_ID for deployment:
1. Clone the [AWS VPC Creator](https://github.com/SofarDatas/aws-vpc-creator) repository.
2. Follow the repository's README to deploy the VPC stack.
3. After successful deployment, find the VPC_ID in the CloudFormation stack outputs.
4. Copy the VPC_ID and use it as the value for the VPC_ID environment variable in your .env file.

Example:
```
VPC_ID=vpc-0123456789abcdef0
```
```